.circleci/config.yml

```diff
@@ -49,16 +49,18 @@ jobs:
           name: Pull Deps
           command: |
             source ${INIT_ENV}
-            ./src/CARMANovatelGpsDriver/docker/checkout.sh ${PWD}
+            ./src/CARMANovatelGpsDriver/docker/checkout.bash -r ${PWD}
```

Comment:
Is this path correct?

Reply:
path is correct

Reply:
that dir that it's running the script in gets created a little higher up in the file